tests/unit/mainwindow/test_prompt.py

```diff
@@ -130,3 +131,23 @@ def test_root_path(self, get_prompt):
         """With / as path, show root contents."""
         prompt = get_prompt('/')
         assert prompt._file_model.rootPath() == '/'
+ 
+    def test_expand_user(self, qtbot, get_prompt, monkeypatch, tmp_path):
+        testdir = tmp_path / 'test'
+        for directory in ['bar', 'bat', 'foo']:
+            (testdir / directory).mkdir(parents=True)
+        
+        homedir = str(testdir)
+        monkeypatch.setenv('HOME', homedir)  # POSIX
+        monkeypatch.setenv('USERPROFILE', homedir)  # Windows
+        print(pathlib.Path.home())
+        assert str(pathlib.Path.home()) == homedir
```

Comment:
Please add a blank line below this to keep it separated from the rest of the test.